### PR TITLE
Implements default fee

### PR DESCRIPTION
**Pull request by jacobburrell**
#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [x ] Tested on iOS Emulator
- [ ] Tested on small Android
- [ ] n/a

### CODE REVIEW

File: src/components/scenes/ChangeMiningFeeScene.js [hunk 1]
```diff
@@ -96,6 +108,17 @@ export class ChangeMiningFee extends React.Component<Props, State> {
           {customFormat != null ? this.renderRadioRow('custom', s.strings.mining_fee_custom_label_choice) : null}
           {customFormat != null ? this.renderCustomFee(customFormat) : null}
           {this.renderFeeWarning()}
+          <View>
```
Comment (jacobburrell) [inline src/components/scenes/ChangeMiningFeeScene.js:111]: Remove extra view

File: src/components/scenes/ChangeMiningFeeScene.js [hunk 2]
```diff
@@ -96,6 +108,17 @@ export class ChangeMiningFee extends React.Component<Props, State> {
...
+            <View style={styles.row}>
```
Comment (jacobburrell) [inline src/components/scenes/ChangeMiningFeeScene.js:112]: Turn into component which receives the props:

-  label/children
- init value
- on change (function)

File: src/components/scenes/ChangeMiningFeeScene.js [hunk 3]
```diff
@@ -196,21 +216,37 @@ const rawStyles = {
...
+      currencySettings: state.ui.settings[getSelectedCurrencyCode(state)],
+      spendTargets: getGuiMakeSpendInfo(state).spendTargets
+    }
+  },
   (dispatch: Dispatch): DispatchProps => ({
-    onSubmit(networkFeeOption: string, customNetworkFee: Object, walletId: string, currencyCode?: string) {
-      dispatch(sendConfirmationUpdateTx({ networkFeeOption, customNetworkFee }, true, walletId, currencyCode))
+    // dispatch = setter
+    onSubmit(networkFeeOption: FeeOption, isDefault: boolean, customFee: JsonObject, walletId: string, currencyCode?: string) {
+      if (isDefault && currencyCode) {
```
Comment (thehobbit85) [inline src/components/scenes/ChangeMiningFeeScene.js:246]: There's a bug here which prevents ever turning off the default fee option.

**Review by thehobbit85 — CHANGES_REQUESTED** [review]
Fix the bug and also make sure that if there is a default fee, you use that when creating a tx and not 'standard'.

File: src/components/scenes/ChangeMiningFeeScene.js (hunk 3 above)
Comment (jacobburrell) [reply src/components/scenes/ChangeMiningFeeScene.js:246]: Fixed and pushed with the setting of default fee to 'none' and checking for that in the case the user toggles the default fee off.

**Pull request closed without merging**